### ISSUE TITLE
API v1 completeness audit

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#1405]
## Description

[//]: # ( A plain language description of the epic. Who, what, why. )

**Who**: Product owner
**What**: Views for Postgrest that present public data for dissemination.
**Why**: Because without the API views, we cannot disseminate our data.

This story is about *creating the API views for Postgrest*, and is not about how we model the collection (Epic #1402) or move data between our collection tables and dissemination tables (Epic #1403).

```[tasklist]
### Stories
- [ ] General has an API view attached to the dissemination schema
- [ ] FederalAward has an API view attached to the dissemination schema
- [ ] Finding has an API view attached to the dissemination schema
- [ ] Passthrough has an API view attached to the dissemination schema
- [ ] Note has an API view attached to the dissemination schema
- [ ] FindingText has an API view attached to the dissemination schema
- [ ] CAPText has an API view attached to the dissemination schema
- [ ] Revision has an API view attached to the dissemination schema
```

**Comment by jadudm** [comment]
#1379 had some notes/comments that may be relevant to this epic.

**Comment by jadudm** [comment]
Closed via #1452.
